dbld/images/required-apt/all.txt

```diff
@@ -44,10 +44,12 @@ strace
 vim
 
 # debian packaging tools
+fakeroot
```

Comment:
If you put a package into the "all.txt" list, than you should:
- check if the package is really available on all platforms (fakeroot is)
- remove it from the distribution specific lists (in this case: trusty)

Reply:
Thanks for the feedback.
I have removed dh-exec and fakeroot from platform specific files.